pkg/rpctype/rpctype.go

```diff
@@ -70,6 +70,8 @@ type HubConnectArgs struct {
 	Calls []string
 	// Current manager corpus.
 	Corpus [][]byte
+	// Current manager repros.
+	Repros [][]byte
```

Comment:
There is no need to send this on connect, handling this just in sync is enough.
For normal inputs we have to send them in connect, so they are not discarded by hub between connect and first sync.